workers/unity/Packages/io.improbable.gdk.core/Utility/EntityTemplate.cs

```diff
@@ -41,6 +41,41 @@ public static string GetWorkerAccessAttribute(string workerId)
         /// </remarks>
         public void AddComponent<TSnapshot>(TSnapshot snapshot)
             where TSnapshot : struct, ISpatialComponentSnapshot
+        {
+            AddComponent((ISpatialComponentSnapshot) snapshot);
```

Comment:
Do we still need the generic functions?

Reply:
Hmm possibly not, it should be backwards compatible to remove the generic ones. Let me have a think

Reply:
Removed the generic `Add` and `Set` variants. The rest have both a generic and non-generic.